src/containers/SubjectPage/components/SubjectPageContent.jsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectT } from 'ndla-i18n';
+import { Breadcrumb } from 'ndla-ui';
+import { toBreadcrumbItems } from '../../../routeHelpers';
+import {
+  GraphQLSubjectPageShape,
+  GraphQLSubjectShape,
+} from '../../../graphqlShapes';
+import { TopicShape } from '../../../shapes';
+import SubjectPageSingle from './layout/SubjectPageSingle';
+import SubjectPageDouble from './layout/SubjectPageDouble';
+import SubjectPageStacked from './layout/SubjectPageStacked';
+import { toTopicMenu } from '../../../util/topicsHelper';
+
+const SubjectBreadCrumb = injectT(
+  ({ t, subject }) =>
+    subject ? (
+      <Breadcrumb
+        items={toBreadcrumbItems(
+          t('breadcrumb.toFrontpage'),
+          subject,
+          undefined,
+          undefined,
+        )}
+      />
+    ) : null,
+);
+
+SubjectBreadCrumb.propTypes = {
+  subject: GraphQLSubjectShape,
+};
+
+const SubjectPageContent = ({ layout, subject, ...rest }) => {
+  const breadcrumb = <SubjectBreadCrumb subject={subject} />;
+  const topics =
+    subject && subject.topics
+      ? subject.topics
+          .filter(
+            topic => !topic || !topic.parent || topic.parent === subject.id,
```

Comment:
Is this nesseary after [this](https://github.com/NDLANO/ndla-frontend/pull/265/files) change?